### ISSUE TITLE
A11y_Azure Tools Storage Explorer_FileShare_CreateNewDirectory_StatusMessage: After we create a new directory/Folder in fileshare, no status message is announced by the screenreader

### ISSUE THREAD

**Issue by Jtith** [microsoft/AzureStorageExplorer#6373]
**GitHub Tags:**
#A11yMAS;#A11yTCS;#Mac;#DesktopApp;#A11ySev3;#BM_AzureToolsStorageExplorer_Mac_Dec2022;#Azure Tools Storage Explorer;#WCAG4.1.3;#Voiceover;#FTP;#DesktopApp;#AINotInScope;

## Actual results:
Screen reader is not announcing the status message when we create any folder or directory. 
## Expected results:
Screen reader should announce the "New folder successfully created"
##Application details
Microsoft Azure Storage Explore
version: 1.27.0

## OS Details
Mac details: macOS Ventura 13.1
Screenreader: Voiceover
## Repro Steps:

1. Launch Storage Explorer
2. Tab to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Tab to "File Shares" of the selected subscription and expand it.
5. Tab to tester and click on it.
6. Turn on voiceover.
7. Tab to New Folder button and click on it.
8. Observe if any status message is announced by screenreader or not.

## User Impact:
Screen reader user will not know about if new folder is created.

## Attachments:

[6373.zip](https://github.com/microsoft/AzureStorageExplorer/files/10243893/6373.zip)

**Comment by Jtith** [comment]
i have verified in the Environment, Issue is no repro. Hence, closing the bug.

<img width="1440" alt="MicrosoftTeams-image (8)" src="https://user-images.githubusercontent.com/100552367/214561122-07d4ec2a-84e8-4242-b11b-6c3eb7f0ad04.png">

**Comment by fsteffi** [comment]
GitHubTags:#Closed;
